4. User Guides/Managing your Account/README.md

```diff
@@ -0,0 +1,98 @@
+# Managing your account
+
+To access and manage your Rocket.Chat account on a server, login and **click** on your **profile picture** or **username** and select **My Account**.
+
+Changing an account setting only changes it for the server you are connected to and not all servers you have access to. You will need to configure your account on each server you connect to.
+
+You can manage settings for the following account sections:
+
+## Preferences
+
+### Localization
+
+Set the language to use for the Rocket.Chat user interface.
+
+### Messages
+
+Enable or disable all desktop notifications, test desktop notifications and configure:
+
+- The global notification duration in seconds
+- Enter key behaviour
+  - Normal mode (send with Enter)
+  - Alternative mode (send with Enter + Ctrl/Alt/Shift/CMD)
+  - Desktop mode (only sends with enter on desktop)
+- View mode
+  - Normal
+  - Cozy
+  - Compact
+- Offline Email Notifications
+  - Disabled
+  - Every Mention/DM
+
+and enable or disable the following notifications:
+
+- Unread Tray Icon Alert
+- Use Emojis
+- Convert ASCII to Emoji
+- Auto Load Images
+- Save Mobile Bandwidth
+- Collapse embedded media by default
+- Unread Rooms Mode
+- Hide usernames
+- Hide roles
+- Hide right side bar with click
+- Hide Avatars
+- Merge private groups with channels
+
+### Highlights
+
+Specify a comma separated list of words or phrases to trigger notifications for when mentioned by anyone in public or private channels you are a member of.
+
+### Sound
+
+Configure audio notifications used for the following events:
+
+- New Room
+- New Message
+
+Available audio notifications:
+
+- None
+- Beep
+- Chelle
+- Ding
+- Droplet
+- Highbell
+- Seasons
+
+## Profile
+
+Verify and edit the following profile information:
+
+- Name
+- Username
+- Email
+- Set a new password
+
+If your email has been verified by the Rocket.Chat server a tick will appear alongside it.
+
+## Avatar
+
+Set or change your avatar for a Rocket.Chat server using one of the options:
+
+- Use your username initials to autogenerate an avatar
+- Upload a files
+- Use a URL
+- Use Gravatar avatar
+- Use GitHub avatar
+- Use GitLab avatar
+- Use Google avatar
+- Use Facebook avatar
+- Use Twitter avatar
+- Use LinkedIn avatar
+
+## Security
+
+Enable or disable Two Factor Authentication (2FA) for your account on a Rocket.Chat server.
```

Comment:
I am not sure either. Let's get this in and then add further 2FA detail later in a 2FA article.